app.js

```diff
@@ -0,0 +1,46 @@
+let books = [];
+
+const booksContainer = document.getElementById('books');
+const addBtn = document.getElementById('add-btn');
+const title = document.getElementById('title');
+const author = document.getElementById('author');
+
+if (JSON.parse(localStorage.getItem('books')).length !== 0) {
+  books = [...books, ...JSON.parse(localStorage.getItem('books'))];
+}
+
+const setItems = () => {
+  booksContainer.innerHTML = '';
+  books.forEach((book) => {
+    const li = document.createElement('li');
+    const h2 = document.createElement('h2');
+    h2.textContent = book.title;
+    const h3 = document.createElement('h3');
+    h3.textContent = book.author;
+    const button = document.createElement('button');
+    button.textContent = 'X';
+    button.addEventListener('click', () => {
+      const newBooks = books.filter((books) => books.title !== book.title);
+      books = [...newBooks];
+      localStorage.setItem('books', JSON.stringify(books));
+      setItems();
+    });
+    li.appendChild(h2);
+    li.appendChild(h3);
+    li.appendChild(button);
+    return booksContainer.appendChild(li);
+  });
+};
+
+addBtn.addEventListener('click', () => {
+  books.push({
+    title: title.value,
+    author: author.value,
+  });
+  localStorage.setItem('books', JSON.stringify(books));
+  title.value = '';
+  author.value = '';
+  setItems();
+});
+
+setItems();
```

Comment:
- [ ] I cannot add any book. Please make sure the function is working properly. 
<img width="398" alt="Capture" src="https://user-images.githubusercontent.com/58504655/133094933-772ea54a-ee91-49ff-962c-268777318ca9.PNG">

- [ ] Please add a proper function to remove a book. 
<img width="564" alt="Capture" src="https://user-images.githubusercontent.com/58504655/133095261-eaab1ead-86d3-4cf5-b295-a7ba4e6eaac4.PNG">

- [ ] Please make sure the data is preserved in the browser's memory by using localStorage.

Reply:
This issue is addressed well. Excellent work! :clap: